docs/Config_Reference.md

```diff
@@ -4466,6 +4466,100 @@ adc2:
 #   above parameters.
 ```
 
+## Load Cells
+
+### [load_cell]
+Load Cell. Uses a Bulk ADC sensor attached to a load cell to create a digital
+scale.
+
+```
+[load_cell]
+sensor_type:
+#   This must be one of the supported Bulk ADC Sensor types
+```
+
+## Bulk ADC Sensors
+
+Klipper includes support for generic Analog to Digital Converter (ADC) chips
+that can sample at a high data rate. These sensors measure a voltage that can be
+converted into various measurements (temperature, force etc.) by other modules.
+These sensors may be used in any config section that accepts a Bulk ADC (such as
+a `[load_cell]`).
```

Comment:
I don't think it is a good idea to expose the "bulk adc" abstraction to end-users.  The loadcell documentation and following hx711/hx717/ads1220 seem fine, but I don't see a reason to complicate things with a "bulk adc" intermediary.  Seems simpler to describe this all as part of the loadcell implementation.  If at some future point we need to describe some other (non-loadcell) chip that uses these sensors then we can tackle how to document that at that later point.

Reply:
As you wish.

I was just trying to highlight that these sensors are not 'load cell sensors'. For anyone new to the codebase its not obvious that they can re-use these if they read the documentation.